### DISCUSSION TITLE
Why is there no beta scale estimate for the transition-state probabilities "1 -> 1" and "2 -> 2" using `CIbeta`?

### DISCUSSION THREAD

**Discussion by Dekike** [bmcclintock/momentuHMM#62] [Q&A] [answered]
Hi Brett,

Exploring `CIreal(example$m, alpha = 0.95, parms = "gamma")`, I observe that there are 4 possible transition-state probabilities:

   1. To remain in the state 1 (`1 -> 1`).
   2. To change from the state 1 to the state 2 (`1 -> 2`).
   3. To change from the state 2 to the state 1 (`2 -> 1`).
   4. To reman in the state 2 (`2 -> 2`).

Why, when I use `CIbeta(example$m)$beta$est`, Do I see only two beta estimates? One for the transition state `1 -> 2` and other for `2 -> 1`. Why there are no beta estimates for the transition states `1 -> 1` and `2 -> 2`?

I am interested in knowing if several covariates affect the transition-state probability of my target species, and I was wondering why there is no beta for the transition states `1 -> 1` and `2 -> 2`.

**Comment by bmcclintock** [comment, marked as answer]
See Section 4.1.3 in the moveHMM [vignette](https://cran.r-project.org/package=moveHMM/vignettes/moveHMM-guide.pdf).
